src/AsyncSessionMiddlewareFactory.php

```diff
@@ -2,32 +2,39 @@
 
 namespace Interop\Session\Middleware\Async;
 
-use Psr\Http\Message\{ServerRequestInterface as Request, ResponseInterface as Response};
-use Interop\Session\Manager\{SessionManagerInterface as SessionManager, Utils\DefaultManager\DefaultSessionManager as DefaultManager};
+use Psr\Http\Message\{
+    ServerRequestInterface as Request, ResponseInterface as Response
+};
+use Interop\Session\Manager\{
+    SessionManagerInterface as SessionManager, Utils\DefaultManager\DefaultSessionManager as DefaultManager
+};
 use Interop\Session\Configuration\SessionConfigurationInterface;
 
 use Interop\Session\SessionInterface;
 
-class AsyncSessionMiddlewareFactory {
+class AsyncSessionMiddlewareFactory
```

Comment:
I used the factory's way of zend-service: https://zendframework.github.io/zend-servicemanager/configuring-the-service-manager/
It has been tested on the zend-expressive bridge of Aura.DI.